src/platform_impl/windows/window.rs

```diff
@@ -386,14 +386,10 @@ impl Window {
         self.thread_executor.execute_in_thread(move || {
             // Change video mode if we're transitioning to or from exclusive
             // fullscreen
-            match (&old_fullscreen, &fullscreen) {
-                (&None, &Some(Fullscreen::Exclusive(ref video_mode)))
-                | (
-                    &Some(Fullscreen::Borderless(_)),
-                    &Some(Fullscreen::Exclusive(ref video_mode)),
-                )
-                | (&Some(Fullscreen::Exclusive(_)), &Some(Fullscreen::Exclusive(ref video_mode))) =>
-                {
+            match (&old_fullscreen, &mut fullscreen) {
+                (None, Some(Fullscreen::Exclusive(video_mode)))
+                | (Some(Fullscreen::Borderless(_)), Some(Fullscreen::Exclusive(video_mode)))
+                | (Some(Fullscreen::Exclusive(_)), Some(Fullscreen::Exclusive(video_mode))) => {
```

Comment:
This could just be:
```suggestion
                (_, Some(Fullscreen::Exclusive(video_mode))) => {
```

Right?

Reply:
ah nice! done